### PR TITLE
Package ISO8601.0.2.6

### PR DESCRIPTION
**Pull request by c-cube**
### `ISO8601.0.2.6`
ISO 8601 and RFC 3999 date parsing for OCaml

---
* Homepage: http://github.com/ocaml-community/ISO8601.ml/
* Source repo: git+https://github.com/ocaml-community/ISO8601.ml.git
* Bug tracker: https://github.com/ocaml-community/ISO8601.ml/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by c-cube** [comment]
I believe the toml tests wouldn't pass on the datakit machines with version 0.2.5 either. I'll work on merging a bugfix PR for 0.3, but 0.2.6 behaves exactly the same as previous versions except it builds with dune.

File: packages/ISO8601/ISO8601.0.2.6/opam [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+bug-reports: "https://github.com/ocaml-community/ISO8601.ml/issues"
+depends: [
+  "dune" {>= "1.0"}
+  "base-unix"
+  "ocaml" {>= "3.12.1"}
+  "odoc" {with-doc}
+  "ounit" {with-test}
+]
+build: [
+  ["dune" "build" "-p" name]
```
Comment (kit-ty-kate) [inline packages/ISO8601/ISO8601.0.2.6/opam]:
```suggestion
  ["dune" "build" "-p" name "-j" jobs]
```

File: packages/ISO8601/ISO8601.0.2.6/opam [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  ["dune" "build" "@doc" "-p" name] {with-doc}
```
Comment (kit-ty-kate) [inline packages/ISO8601/ISO8601.0.2.6/opam]:
```suggestion
  ["dune" "build" "@doc" "-p" name "-j" jobs] {with-doc}
```

**Comment by kit-ty-kate** [comment]
Otherwise it looks good

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**